/**/odom_map_remapper:
  ros__parameters:
    odom_topic: sim_ground_truth_pose
    tf_topic: tf
    map_frame: map
    odom_frame: odom
    robot_frame: base_link